<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>201</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>133</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="show_sig">
          <property name="text">
           <string>Show Signal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn2">
          <property name="text">
           <string>source: sin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn3">
          <property name="text">
           <string>refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn4">
          <property name="text">
           <string>animation: stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QOpenGLWidget" name="gl_widget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>651</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="go_page1">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>go 1st page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="go_page2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>go 2nd page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Port 1</string>
      </property>
      <widget class="QPushButton" name="back_main1">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>40</y>
         <width>95</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Main Page</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>66</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sin 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>66</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sin 2</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinSin1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>90</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinSin2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>90</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Port 2</string>
      </property>
      <widget class="QPushButton" name="back_main2">
       <property name="geometry">
        <rect>
         <x>364</x>
         <y>36</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Main Page</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinCos1">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>90</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinCos2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-360</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>90</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>66</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cos 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>66</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cos 2</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
